.github/workflows/fabric_acctest.yml

```diff
@@ -197,31 +215,26 @@ jobs:
         run: |
           go test -json ./... -v -coverprofile coverage_pfcr.txt -covermode=atomic -count 1 -parallel 8 -run "(PFCR)" -timeout 180m | tee pfcr_test_output.json
 
-      - name: Upload PFCR Testing Log
-        if: ${{ always() }}
-        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4
-        with:
-          name: pfcr_test_logs
-          path: pfcr_test_output.json
-
-      - name: Sweeper PFCR
-        if: ${{ always() }}
+      - name: TF Fabric PFCR acceptance tests STS creds
+        timeout-minutes: 180
```

Comment:
The Fabric acceptance test workflow was recently updated so that the [TF Fabric PFCR acceptance tests](https://github.com/equinix/terraform-provider-equinix/actions/runs/17472658879/job/49624787652) step fails if the acceptance tests fail, which means that the STS creds test step is now being skipped:  https://github.com/equinix/terraform-provider-equinix/actions/runs/17472658879/job/49624787652

To allow the STS creds tests to run even if the other tests fail, you would need to add an if statement to this step.  Specifying `if: always()` is the fastest way to do that, but you could consider using a more specific conditional so that the STS creds tests only run if the `get_id_token` step succeeded.

Reply:
An adjustment to my earlier suggestion:  rather than modifying how this test step runs, I recommend copying the `test-PFCR` job in its entirety, renaming it to `test-STS-creds` (or similar) and updating the acceptance test step to run the intended tests.  If/when all tests are able to run using STS token exchange, the Fabric team can choose to update and/or consolidate the test jobs as needed.